keep/api/core/db.py

```diff
@@ -193,6 +193,19 @@ def try_create_single_tenant(tenant_id: str) -> None:
         except Exception:
             pass
 
+    # migrating presets table
+    with Session(engine) as session:
+        try:
+            logger.info("Migrating Preset table")
+            session.exec(
+                "ALTER TABLE preset ADD COLUMN is_private BOOLEAN NOT NULL DEFAULT 0;"
+            )
+            session.exec("ALTER TABLE preset ADD COLUMN created_by TEXT DEFAULT '';")
```

Comment:
I'm getting this: `Error Code: 1101. BLOB, TEXT, GEOMETRY or JSON column 'created_by' can't have a default value`

Reply:
```suggestion
            session.exec("ALTER TABLE preset ADD COLUMN created_by VARCHAR(1024) DEFAULT '';")
```